packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -193,7 +193,9 @@ export const loadCallPageWithParticipantVideos = async (pages: Page[]): Promise<
       // The tests often timeout at this step because loading remote video streams can take > 5 seconds.
       // Extend the timeout here to trade flakiness for potentially longer test runtime.
       // Test flake has a much larger impact on overall CI time than individual test runtime.
-      { timeout: 2 * perStepLocalTimeout() }
+      //
+      // The extended timeout was determined by stress testing on CI.
```

Comment:
https://github.com/Azure/communication-ui-library/runs/7679141230

This timeout was the only one that was hit. It was hit *60 times* in 7 runs of the entire suite :(